RELEASE-NOTES.md

```diff
@@ -1,5 +1,41 @@
 # Prefect Release Notes
 
+## Release 2.7.2
+
+### Extensions to the new pausing feature!
+
+
+See https://github.com/PrefectHQ/prefect/pull/7738 and https://github.com/PrefectHQ/prefect/pull/7863 for details
```

Comment:
Super optional: We could add more info before referencing the PRs.
```suggestion
This release extends the recently added pause/resume functionality with non-blocking functionality. You can now pass the `reschedule` argument to the `pause_flow_run` utility to gracefully stop the infrastructure process and put the flow run back into a `Scheduled` state. Similarly, you can pause the run from the friendly _Pause_ button directly from the flow run UI page. See https://github.com/PrefectHQ/prefect/pull/7738 and https://github.com/PrefectHQ/prefect/pull/7863 for details
```

Reply:
@anticorrelator is going to add a full blurb here.